spec.html

```diff
@@ -33428,9 +33428,12 @@ <h1>String.prototype.lastIndexOf ( _searchString_ [ , _position_ ] )</h1>
           1. If _numPos_ is *NaN*, let _pos_ be +&infin;; otherwise, let _pos_ be ! ToIntegerOrInfinity(_numPos_).
           1. Let _len_ be the length of _S_.
           1. Let _start_ be the result of clamping _pos_ between 0 and _len_.
+          1. If _searchStr_ is the empty String and _start_ &le; _len_, return 𝔽(_start_)_.
```

Comment:
`_start_ &le; _len_` always holds, because we've constrained it to be so on the previous line. So just

```suggestion
          1. If _searchStr_ is the empty String, return 𝔽(_start_)_.
```

(This line isn't actually necessary, nor the similar line in `StringIndexOf`, but it seems fine to keep it.)